### PR TITLE
Small fixes for knowledge panels

### PR DESCRIPTION
**Pull request by stephanegigandet**
- Fixed some icons paths (the Eco-Score ones existed in prod but not in github)
- Using the result of a FILTER to set a variable in Template::Toolkit works in the dev server, but not in production and in the docker environment. Now exposing the sprintf function directly.
- Fixed the display of packaging components by recycling types
- Fixed the display of environmental labels panels when we don't have an Eco-Score (no Agribalyse category) but when we have organic labels.
- Now showing the panels in the product page for all moderators, in order to uncover issues before we set it for all users.

### CODE REVIEW

File: templates/api/knowledge-panels/environment/carbon_footprint.tt.json [hunk 1]
```diff
@@ -16,9 +16,9 @@
     "evaluation": "good",
     [% END %]
     "title_element": {
-        [% SET driving_100g_rounded = driving_100g FILTER format('%.1f'); %]
+        [% SET driving_100g_rounded = sprintf('%.1f', driving_100g) %]
         "title": "[% f_lang('f_equal_to_driving_km_in_a_petrol_car',  { 'kilometers' => driving_100g_rounded } ) %]",
-        [% SET co2_100g_rounded = co2_100g * 1000 FILTER format('%.0f'); %]
+        [% SET co2_100g_rounded = co2_100g * 1000 FILTER format('%.0f') %]
```
Comment (alexgarel) [inline templates/api/knowledge-panels/environment/carbon_footprint.tt.json:21]: Why didn't you use sprintf here also ?

File: templates/web/pages/product/product_page.tt.html [hunk 2]
```diff
@@ -184,7 +184,7 @@ <h4>[% lang("nova_groups_s") %]
                 <a href="/nova" title="[% a_title %]">
                 [% display_icon('info') %]</a>
             </h4>
-            <a href="/nova" title="[% a_title %]"><img src="/images/misc/nova-group-[% group %].svg" alt="[% display %]" style="margin-bottom:1rem;max-width:100%"></a><br>
+            <a href="/nova" title="[% a_title %]"><img src="/images/attributes/nova-group-[% group %].svg" alt="[% display %]" style="margin-bottom:1rem;max-width:100%"></a><br>
```
Comment (alexgarel) [inline templates/web/pages/product/product_page.tt.html:187]: we are not very consistent… in donate_banner.tt we use static.openfoodfacts.org, there no. (not so important, but just to notice)

**Review by alexgarel — APPROVED** [review]
LGTM appart from two small comments.

File: templates/api/knowledge-panels/environment/carbon_footprint.tt.json (hunk 1 above)
Comment (stephanegigandet) [reply templates/api/knowledge-panels/environment/carbon_footprint.tt.json]: Good catch, fixed.

File: templates/web/pages/product/product_page.tt.html (hunk 2 above)
Comment (stephanegigandet) [reply templates/web/pages/product/product_page.tt.html:187]: Right, we should fix that in another PR.

**Pull request merged**